src/ui/Combobox/Combobox.ts

```diff
@@ -108,7 +107,17 @@ export class Combobox implements ICombobox {
     this.input.updateAccessibilityAttributes(attributes);
   }
 
-  public updateAriaLive(text: string) {
+  public updateAriaLive() {
+    if (!this.values.hasValues()) {
+      this.options.ariaLive.updateText(l('NoValuesFound'));
+      return;
+    }
+
+    let text = l('ShowingResultsWithQuery', this.values.numberOfValues, this.input.value, this.values.numberOfValues);
```

Comment:
hmm, is the combobox announcement referring to search results or query suggestions here?

Reply:
Combobox shows results for a value, not necessary to have something more specific IMO.
```
"ShowingResults": {
  "en": "{0} result<pl>s</pl>",
  "fr": "{0} résultat<pl>s</pl>"
},
"ShowingResultsWithQuery": {
  "en": "{0} result<pl>s</pl> for {1}",
  "fr": "{0} résultat<pl>s</pl> pour {1}"
},
```

Reply:
If I am a blind user typing into the combobox, would it be clear that there query suggestions that I can explore?
Versus when in search-as-you-type mode, where typing into the combobox pulls up new search results?